blueocean-dashboard/src/main/js/creation/bitbucket/cloud/BbCloudFlowManager.js

```diff
@@ -362,21 +375,45 @@ export default class BbCloudFlowManager extends FlowManager {
             this._logEvent(event);
         }
 
-        if (event.job_multibranch_indexing_status === 'INDEXING' && event.job_run_status === 'QUEUED') {
-            this.queuedIndexAllocations++;
-        }
-
-        if (event.job_multibranch_indexing_result) {
-            this.queuedIndexAllocations--;
+        if (event.jenkins_object_type === 'org.jenkinsci.plugins.workflow.job.WorkflowRun') {
```

Comment:
Should there be some additional logic here to ensure the if block is only entered when the event's properties correspond to the expected name of the pipeline? Otherwise this code may get executed multiple times if the server is running other pipelines at the same time?